testfiles/FurnaceWithDXSystem.idf

```diff
@@ -1368,6 +1368,8 @@
 
   Output:Variable,*,Unitary System Fan Part Load Ratio,timestep;
 
+  Output:Variable,*,Fan Electricity Rate,detailed; !- HVAC Average [W]
```

Comment:
If this variable needs to stay in then it should be switched to a `timestep` interval to match the other output variables, otherwise the output csv becomes difficult to read.  I think the PR is otherwise ready to go, so I'll make that quick change and merge if so.